### DISCUSSION TITLE
Control AirPlay playback from iOS devices

### DISCUSSION THREAD

**Discussion by heccoder** [music-assistant/support#3185] [Feature requests and ideas]
Currently when playback is initiated from MA to a HomePod, the speaker disappears from the iOS Control Center. This prevents controlling playback or seeing what is being played from other devices. If I start playback from another system like Sonos, I can control it from iOS.  In the video you can see how "Office HomePod" disappears when playback starts in MA and reappears immediately after the speaker is removed from the playback.
Is this a limitation of AirPlay 1? 

https://github.com/user-attachments/assets/388e4053-e3ba-46ca-a778-e74485d24275

**Comment by Schlichi83** [comment]
The same issue is already discussed here:
https://github.com/orgs/music-assistant/discussions/1561

This is the number 1 issue when using HomePods, Apple Music and Music Assistant.

**Reply by JDogg016** [reply]
I guess the issue is whether this is possible (ie controlling MA playback from HomePod or iOS device)?

If possible, is there a feature poll where we could gauge community interest in this?

**Reply by OzGav** [reply]
You upvote these posts and we can see the interest

**Comment by MacSergey** [comment]
I would like to have this feature implemented. I have a lot of Apple devices and it is so convenient to be able to control music from whatever device you found first. I started using MA and I really miss this feature
